its/tests/src/test/java/its/SonarCloudTests.java

```diff
@@ -471,6 +472,7 @@ void testConnection() throws ExecutionException, InterruptedException {
   }
 
   @Test
+  @Disabled("Because the the feature is intended for SQ and the IT was only added 'on-top' but is too flaky!")
```

Comment:
Can't we simply drop it then? I am a bit worried by all those @disabled tests.

Reply:
I'm in favor, I see SLCORE already has some ignored / disabled tests. At some point we should get rid of the rest too.

Reply:
Done